src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/controller/PartnershipController.java

```diff
@@ -0,0 +1,30 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.controller;
+
+import jakarta.servlet.http.HttpServletRequest;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import uk.gov.companieshouse.limitedpartnershipsapi.model.dto.LimitedPartnershipSubmissionCreatedResponseDto;
+import uk.gov.companieshouse.limitedpartnershipsapi.utils.ApiLogger;
+
+import java.net.URI;
+
+import static uk.gov.companieshouse.limitedpartnershipsapi.utils.Constants.URL_PARAM_TRANSACTION_ID;
+
+@RestController
+@RequestMapping("/transactions/{" + URL_PARAM_TRANSACTION_ID + "}/limited-partnership/partnership")
+public class PartnershipController {
+
+    @PostMapping
+    public ResponseEntity<Object> createPartnership(
+            HttpServletRequest request) {
+
+        ApiLogger.debug("createPartnership");
+
+        URI location = URI.create("/transactions/12321123/limited-partnership/partnership/3235233232");
```

Comment:
If this is temporary for testing purposes is this value something we could pass in thought the request body rather than hardcoding? Do we need this?

Reply:
That's right, the next PR(s) to complete this story will have a proper request body. These are only hardcoded to provide an endpoint for others to use when working on their own tickets in the meantime.